cosipy/response/FullDetectorResponse.py

```diff
@@ -190,21 +190,24 @@ def _open_rsp(cls, filename, Spectrumfile=None,norm="Linear" ,single_pixel = Fal
                 elif key == 'SA':
                     area_sim = float(line[1])
                     
-                elif key == "SP" and line[1]!="true" and line[1]!="false" :
-                    norm = str(line[1])
+                elif key == "SP" :
                     
+                    try :
+                        norm = str(line[1])
+                    except :
+                        print(f"norm not found in the file ! We assume {norm}")
+
                     if norm =="Linear" :
                         emin = int(line[2])
                         emax = int(line[3])
                         
-                elif key == "SP" and line[1]=="true" :
-                    sparse = True
-                    
-                elif key == "SP" and line[1]=="false" :
-                    sparse = False
+             
 		
                 elif key == "MS":
-                    sparse = bool(line[1])	
+                    if line[1] == "true" :
```

Comment:
Was `bool('true')`/ `bool('false')`  failing? I'd had thought it should work. Or what was the issue?

Reply:
Yeah me too but it is actually always return `True` , even bool("false")

Reply:
Right! I thought I had checked, but saw it wrong. Never mind, thanks.